cmakebuild.js

```diff
@@ -24,7 +24,7 @@ function runCmake (opts, target, cb) {
       }
     }
 
-    var proc = spawn(cmakeJsPath, args)
+    var proc = spawn(cmakeJsPath, args, process.platform === "win32" ? { shell: true } : undefined)
```

Comment:
```suggestion
    var proc = spawn(cmakeJsPath, args, process.platform === 'win32' ? { shell: true } : undefined)
```

Reply:
@kraenhansen seems likes last step to get it in :-)

Reply:
thanky ou @lovell